app/helpers/blacklight/facets_helper_behavior.rb

```diff
@@ -151,33 +152,21 @@ def path_for_facet(facet_field, item)
   # @see #render_facet_value
   # @param [Blacklight::Solr::Response::Facets::FacetField] facet_field
   # @param [String] item
-  def render_selected_facet_value(facet_field, item)
-    remove_href = search_action_path(search_state.remove_facet_params(facet_field, item))
-    content_tag(:span, class: "facet-label") do
-      if @locations[item.value]
-
-        content_tag(:span, facet_display_value(facet_field, @locations[item.value]["name"]), class: "selected") +
-        # remove link
-        link_to(remove_href, class: "remove") do
-        content_tag(:span, '', class: "glyphicon glyphicon-remove") +
-          content_tag(:span, '[remove]', class: 'sr-only')
-      end
-
-     else
```

Comment:
Same as above.  It came with a syntax error?! Didn't put a lot of thought into what this was doing and copied from Discovery.

Reply:
I think I added this because Blacklight assumed displayable text as facet values, and we needed to map short codes to displayable text in some cases... it's been a long time, but I think that's what this is for.